src/main/java/emu/grasscutter/game/props/PlayerProperty.java

```diff
@@ -60,7 +60,15 @@ public enum PlayerProperty {
     PROP_IS_AUTO_UNLOCK_SPECIFIC_EQUIP(10044), // New; unknown/un-used.
     PROP_PLAYER_GCG_COIN(10045), // New; unknown/un-used.
     PROP_PLAYER_WAIT_SUB_GCG_COIN(10046), // New; unknown/un-used.
-    PROP_PLAYER_ONLINE_TIME(10047); // New; unknown/un-used.
+    PROP_PLAYER_ONLINE_TIME(10047), // New; unknown/un-used.
+    PROP_PLAYER_ANEMOCULUS(10048, 0), // Amount of Anemoculus collected [0, 65]
+    PROP_PLAYER_ANEMOCULUS_LEVEL(10049, 1, 10), // level of Anemoculus collected [1, 10]
+    PROP_PLAYER_GEOCULUS(10050, 0), // Amount of Geoculus collected [0, 131]
+    PROP_PLAYER_GEOCULUS_LEVEL(10051, 1, 10), // level of Geoculus collected [1, 10]
+    PROP_PLAYER_ELECTROCULUS(10052, 0), // Amount of Electroculus collected [0, 180]
+    PROP_PLAYER_ELECTROCULUS_LEVEL(10053, 1, 10), // level of Electroculus collected [1, 10]
+    PROP_PLAYER_DENDROCULUS(10054, 0), // Amount of Dendroculus collected [0, 270]
+    PROP_PLAYER_DENDROCULUS_LEVEL(10055, 1, 10); // level of Dendroculus collected [1, 10]
```

Comment:
are these actual player properties? if so, they should be considered in the virtual currency section of adding items to the player

Reply:
bigger issue, this aren't props found in-game, these should be relocated elsewhere

Reply:
Ok so I will add these props to another field, something like cityLevelInfo

Reply:
PlayerProgress might be a good place to put them.
There you can probably put them into a map of the type CityInfo, for example like this:
```java
@Data
public class CityInfo {
    int cityId;
    int level = 0;
    int currentProgress = 0
}
```